style.css

```diff
@@ -3,122 +3,198 @@
   overflow-x: hidden;
   padding: 0;
   margin: 0;
+  font-family: "Poppins", sans-serif;
 }
-.mobile-bar{
-  display:  flex;
-  flex-direction:row;
+
+.mobile-bar {
+  padding-top: 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  list-style: none;
+}
+
+.time {
+  margin-right: 280px;
+  margin-left: 20px;
+  align-items: flex-start;
+}
+
+.wifi {
+  margin-right: 12px;
+}
+
+.cellular {
+  margin-right: 12px;
+}
+
+.nav {
+  padding: 20px 15px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.my-logo {
+  font-style: normal;
+  font-weight: bold;
+  line-height: 1.3rem;
+  text-decoration: none;
+  font-size: 19px;
+  color: #6070ff;
```

Comment:
- [Optional] Kindly remove the default behavior of `a` tag, the logo inn the Figma hasn't been underlined so please stick to the Figma 👌👍